jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/TestAppContainer.java

```diff
@@ -61,28 +61,6 @@ public TestAppContainer withUserAuth(String login, String pwd) {
     return this;
   }
 
-  /**
-   * Configures app container for host-to-container access, port will be used as-is from host to
-   * work-around JMX in docker. This is optional on Linux as there is a network route and the
-   * container is accessible, but not on Mac where the container runs in an isolated VM.
-   *
-   * @param port port to use, must be available on host.
-   * @return this
-   */
-  @CanIgnoreReturnValue
-  public TestAppContainer withHostAccessFixedJmxPort(int port) {
-    // To get host->container JMX connection working docker must expose JMX/RMI port under the same
```

Comment:
[for reviewer] this is no longer necessary as we only rely on container-to-container communication and we don't have to use docker NAT to access the JMX endpoint.